src/gen-xml.ts

```diff
@@ -629,26 +633,41 @@ function slideObjectToXml(slide: PresSlide | SlideLayout): string {
-					slideItemObj.options.shadow.offset = valToPts(slideItemObj.options.shadow.offset || 4)
-					slideItemObj.options.shadow.angle = Math.round((slideItemObj.options.shadow.angle || 270) * 60000)
-					slideItemObj.options.shadow.opacity = Math.round((slideItemObj.options.shadow.opacity || 0.75) * 100000)
-					slideItemObj.options.shadow.color = slideItemObj.options.shadow.color || DEF_TEXT_SHADOW.color
-
+				// EFFECTS
+				if (hasEffects) {
 					strSlideXml += '<a:effectLst>'
-					strSlideXml += '<a:' + slideItemObj.options.shadow.type + 'Shdw'
-					strSlideXml += slideItemObj.options.shadow.type === 'outer' ? ' sx="100000" sy="100000" kx="0" ky="0" algn="bl" rotWithShape="0"' : ''
-					strSlideXml += ' blurRad="' + slideItemObj.options.shadow.blur + '"'
-					strSlideXml += ' dist="' + slideItemObj.options.shadow.offset + '" dir="' + slideItemObj.options.shadow.angle + '">'
-					strSlideXml += '<a:srgbClr val="' + slideItemObj.options.shadow.color + '">'
-					strSlideXml += '<a:alpha val="' + slideItemObj.options.shadow.opacity + '"/></a:srgbClr>'
-					strSlideXml += '</a:' + slideItemObj.options.shadow.type + 'Shdw>'
+
+					// EFFECTS > SHADOW REF: @see http://officeopenxml.com/drwSp-effects.php
+					if (slideItemObj.options.shadow && slideItemObj.options.shadow.type !== 'none') {
+						slideItemObj.options.shadow.type = slideItemObj.options.shadow.type || 'outer'
+						slideItemObj.options.shadow.blur = valToPts(slideItemObj.options.shadow.blur || 8)
+						slideItemObj.options.shadow.offset = valToPts(slideItemObj.options.shadow.offset || 4)
+						slideItemObj.options.shadow.angle = Math.round((slideItemObj.options.shadow.angle || 270) * 60000)
+						slideItemObj.options.shadow.opacity = Math.round((slideItemObj.options.shadow.opacity || 0.75) * 100000)
+						slideItemObj.options.shadow.color = slideItemObj.options.shadow.color || DEF_TEXT_SHADOW.color
+
+						strSlideXml += '<a:' + slideItemObj.options.shadow.type + 'Shdw'
+						strSlideXml += slideItemObj.options.shadow.type === 'outer' ? ' sx="100000" sy="100000" kx="0" ky="0" algn="bl" rotWithShape="0"' : ''
+						strSlideXml += ' blurRad="' + slideItemObj.options.shadow.blur + '"'
+						strSlideXml += ' dist="' + slideItemObj.options.shadow.offset + '" dir="' + slideItemObj.options.shadow.angle + '">'
+						strSlideXml += '<a:srgbClr val="' + slideItemObj.options.shadow.color + '">'
+						strSlideXml += '<a:alpha val="' + slideItemObj.options.shadow.opacity + '"/></a:srgbClr>'
+						strSlideXml += '</a:' + slideItemObj.options.shadow.type + 'Shdw>'
+					}
+
+					// EFFECTS > BLUR REF: @see http://officeopenxml.com/drwPic-effects.php
+					// Blur grow has to be applied to the bounding shape
+					// But should minimize compounding the blur effect on the image
+					if (imgBlur?.grow) {
+						imgBlur.radius = valToPts(imgBlur.radius / 10 || 8)
+						strSlideXml += `<a:blur rad="${imgBlur.radius}" grow="1"/>`
```

Comment:
So if I understand correctly, we apply blur to both <blip> and <effectLst> correct?

Reply:
For the grow effect, yes